gearcatalog/frontend/src/components/gear/EditGearForm.tsx

```diff
@@ -34,7 +35,7 @@ match: { params } }: EditGearProps) => {
     useInput(selectedGear.locking);
 
   useEffect(() => {
-    selectGear(params.id, token);
+    selectGear(params.id.toString(), token);
```

Comment:
Why string-ify the id?

Reply:
selectGear takes a string in the first argument.

Reply:
Why? If the first argument is the ID and IDs are numbers, shouldn't it be a number?

Reply:
Yeah, that makes sense. I'll change the type on the functions.